cypress/e2e/data-import/settings/attach-remove-mapping-profile-to-action-profile.cy.js

```diff
@@ -28,6 +28,7 @@ describe('ui-data-import: Attach/Remove a field mapping profile to an action pro
 
   before(() => {
     cy.loginAsAdmin();
+    cy.getAdminToken();
```

Comment:
why need here token?

Reply:
yeap, I deleted profiles by API in the after block